### ISSUE TITLE
Move to https://github.com/emacs-lsp/lsp-clangd

### ISSUE THREAD

**Issue by MaskRay** [tee3/lsp-clangd#3]

**Comment by MaskRay** [comment]
@vibhavp

**Comment by vibhavp** [comment]
Is the author okay with this?

**Comment by tee3** [comment]
I'd be fine with this.  What needs to happen for this?

**Comment by vibhavp** [comment]
@tee3 I've given you write permissions to https://github.com/emacs-lsp/lsp-clangd. You should be able to move the code to the repo.

**Comment by tee3** [comment]
@vibhavp I tried to "transfer" the repository to emacs-lsp, but because I already created it there I can't transfer.  Can you maybe delete and recreate the emacs-lsp/lsp-clangd repository so I can try it again?  If there's a better way, let me know.  Thanks!

**Comment by innerout** [comment]
What has happened and the transfer of the repo is not completed after 2 months? @tee3 @vibhavp

**Comment by tee3** [comment]
I can't transfer the entire repository as it stands now.  I could create a new one over there instead of actually transferring the whole thing.  I'm just waiting for some direction from @vibhavp .

**Comment by innerout** [comment]
Do you have commit permissions?

**Comment by tee3** [comment]
I just pushed my master branch to emacs-lsp/lsp-clangd.  I couldn't transfer it since the repository already existed there and I don't have permission to replace it.  Therefore, tags, releases, branches, issues and pull requests were not transferred and I still have the main repository.

I doubt that's what anyone wanted, but it's up there now.  If anyone wants it to be better, it will have to be driven from the emacs-lsp organization.

The error message I got when trying to transfer:

`emacs-lsp/lsp-clangd already exists and You don’t have the permission to create repositories on emacs-lsp`
